src/pages/community.astro

```diff
@@ -150,37 +154,26 @@ import nixosFoundationLogo from "../assets/image/nixos-foundation-logo.svg";
     <h3 class="text-3xl font-bold font-heading text-nix-blue mt-4">Official spaces</h3>
     <p class="text-gray-700">
       These spaces are monitored and moderated by the NixOS Moderation Team.
+        
+      <ul class="grid grid-cols-1 md:grid-cols-3 gap-4 mt-4 flex-wrap">
+        {
+          platformsMain.map((platform) => (
+            <OfficialCommunityLink
+              href={platform.href}
+              name={platform.name}
+              iconName={platform.iconName}
+              description={platform.description} />
+          ))
+        }
+      </ul>    
     </p>
-    <div class="grid md:grid-cols-3 gap-4 mt-4">
-        <ul class="flex gap-4 flex-col md:col-span-2">
-          {
-            platformsMain.map((platform) => (
-              <CommunityLink
-                href={platform.href}
-                name={platform.name}
-                iconName={platform.iconName}
-                description={platform.description} />
-            ))
-          }
-        </ul>
-        <ul class="flex md:flex-col gap-4 flex-wrap">
-          {
-            platformsOther.map((platform) => (
-              <CommunityLink
-                href={platform.href}
-                name={platform.name}
-                iconName={platform.iconName}
-                type="mobile_icon_desktop_name" />
-            ))
-          }
-        </ul>
-    </div>
```

Comment:
This removes the `platformsOther`, since those options are (for now) merged into `platformsOutside`.

This also makes the DOM mirror what we do for the unofficial links, instead of the older solution which was done to make a 2-3 layout on mobile--we just do a single column now.